build.sbt

```diff
@@ -2,7 +2,7 @@ lazy val root = (project in file(".")).enablePlugins(BuildInfoPlugin).settings(
 
   name := "jacoco4sbt",
   organization := "de.johoop",
-  version := "2.2.0",
+  version := "2.3.0",
```

Comment:
Can you set it to 2.3.0-SNAPSHOT instead? We should use snapshot versions during development. Then the person who cuts a new release should remove the snapshot suffix. 

Also, it looks like there's other places you need to update. See https://github.com/sbt/jacoco4sbt/commit/cd67a62aff8e37a8c60f68a04526c9480137027d. But it'd be really nice if you were able to find a way to update it to only need to be set in one place. Maybe for an example see https://github.com/typesafehub/sbteclipse/blob/master/src/sbt-test/sbteclipse/02-contents/project/plugins.sbt